modern/public/logo.svg

```diff
@@ -12,8 +12,8 @@
  </metadata>
  <g id="layer1">
   <rect id="rect3778" height="64" width="236.1" y="0" x="0" fill="none"/>
-  <ellipse id="path3038" rx="28.995" ry="28.995" transform="rotate(-30)" cy="43.713" cx="11.713" stroke-width="10.699" fill="#fff"/>
-  <g fill="#336">
+  <ellipse id="path3038" rx="28.995" ry="28.995" transform="rotate(-30)" cy="43.713" cx="11.713" stroke-width="10.699" fill="none"/>
+  <g fill="#fff">
```

Comment:
Can we find a way to reuse the same logo and tint it with CSS or something like that?

Reply:
Since our logo file is very small, I have created the Logo component which return svg with fill and stroke we have passed.

Reply:
I noticed that the content of the circular part of the logo is now completely white.

Reply:
Is there a better way to tint logo as an SVG image? The problem with a component is that a lot of users want to change it and keeping it as a static image makes it much easier.

Reply:
There is no way to change fill color using CSS if we render the SVG using img src.

Reply:
I think it is better to use two separate logo files if we are not going with the component.

Reply:
I did a bit of research and it looks like it's possible to tint SVG images using CSS. We just need to assign unique ids to elements inside SVG.